apps/omg_bus/lib/omg_bus/event.ex

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Bus.Event do
```

Comment:
Applied. The only thing I don't like about this is `OMG.Bus.subsribe`. I need to have constants for root chain and child chain prefixes there and in `OMG.Bus.Event`. If you have an idea how to keep them in one place, please share it.

Reply:
I see. I proposed a change to ditch atoms (root chain and child chain prefixes) from the `Event`. So it will looks like normal data passed to the generic function.

> keep them in one place
We're replacing `topic -> {origin, topic}` and trying to not limit `Event` to handle `origins` other than these two. Does it make sense?